Networking/Networking/Remote/BlazeRemote.swift

```diff
@@ -55,6 +64,15 @@ public protocol BlazeRemoteProtocol {
 ///
 public final class BlazeRemote: Remote, BlazeRemoteProtocol {
 
+    public func createCampaign(_ campaign: CreateBlazeCampaign,
+                               siteID: Int64) async throws -> Int64 {
+        let path = Paths.campaigns(siteID: siteID)
+        let parameters = try campaign.toDictionary(keyEncodingStrategy: .convertToSnakeCase)
```

Comment:
We should set `dateFormatter` like how we did with `fetchForecastedImpressions`.

Reply:
Thanks for the suggestion. Updated in https://github.com/woocommerce/woocommerce-ios/pull/11617/commits/e261f537d0c791f4d9a77f65122b26c1a9ca28d6